### ISSUE TITLE
Submit content to Crossref

### ISSUE THREAD

**Issue by hdrury1** [microsimulation/ijm#28]
When Matteo has given us the green light for launch. For now, Matteo is happy that this will not be automated.

**Comment by gnott** [comment]
As a first test, I generated a single Crossref deposit XML file including the 196 articles I have from a year ago. I checked it with Crossref's validation page (https://data.crossref.org/reports/parser.html), and there were no erorrs:

```
Parsing is complete, if no errors were printed the file is valid and well formed
196 DOIs found in file
```

There are some essential values I need for the configuration file, these are what I used:

```
registrant: IJM
depositor_name: IJM
email_address: ijm@example.org
doi_pattern: http://microsimulation.pub/articles/{manuscript}
```

If I can get the correct values for these when I generate the final file, that would be much appreciated!

Is the final website going to be the above URL, and will it be `https://` enabled?

I will also have a couple clarification questions, such as whether I should leave in this for IJM deposits:

```xml
<archive_locations>
	<archive name="CLOCKSS"/>
</archive_locations>
```

When I have all the final XML file I can generate the batch and it can be reviewed for any additional tweaks, perhaps with the help of @FAtherden-eLife.
